### PR TITLE
mas ingestion

### PR DESCRIPTION
**Pull request by southeo**
- Script to ingest MASs
- Small fixes in MASs

### CODE REVIEW

File: mas-ingestion/mas_ingestion.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+    target_filter = {
+        HAS_IDENTIFIER: ['*'],
+        "$['dwc:basisOfRecord']": ['*'],
+        ODS_TYPE: [SPECIMEN_TYPE]
+    }
+    batching = False
+    environment = []
+    secrets = []
+    return build_attributes(name, description, image, tag, target_filter,
+                            batching, environment, secrets)
```
Comment (TomDijkema) [inline mas-ingestion/mas_ingestion.py:115]: Remove

File: mas-ingestion/mas_ingestion.py [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    tag = 'latest'
+    target_filter = {
+        HAS_IDENTIFIER: ['*'],
+        ODS_TYPE: [SPECIMEN_TYPE]
+    }
+    batching = False
+    environment = []
+    secrets = [
+        build_secret('API_USER', 'bold-api-user'),
+        build_secret('API_PASSWORD', 'bold-api-password')
```
Comment (TomDijkema) [inline mas-ingestion/mas_ingestion.py:61]: These passwords are not private aren't they?

File: mas-ingestion/mas_ingestion.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    name = 'mindat-georeferencing'
+    description = 'Uses the Mindat georeferencing API to add Georeference coordinates'
+    image = 'public.ecr.aws/dissco/mindat-georeferencing'
+    tag = 'latest'
+    target_filter = {
+        HAS_EVENT: ['*'],
+        ODS_TYPE: [SPECIMEN_TYPE]
+    }
+    batching = True
+    environment = []
```
Comment (TomDijkema) [inline mas-ingestion/mas_ingestion.py:176]: Its empty a lot, maybe make it an optional parameter?

File: mas-ingestion/mas_ingestion.py (hunk 2 above)
Comment (southeo) [reply mas-ingestion/mas_ingestion.py:61]: just the name of the secret, not the secret itself :)

**Review by samleeflang — APPROVED** [review]

**Pull request merged**